app/main/posts/collections/mode-context.html

```diff
@@ -112,7 +112,7 @@ <h1 class="mode-context-title">
         <!--// END IF //-->
         <div class="form-field">
             <!-- mode context bar is only available in the map so it should take you back there -->
-            <a ng-click="clearFilters()" type="button" class="button-link">
+            <a role="button" tabindex="0" ng-click="clearFilters()" type="button" class="button-link">
```

Comment:
Even though we have the class "button" here (which is a mistake we made when choosing the class-name), this element looks like a link and behaves like one so we should not add the role="button" here. Also, could you please test this works when tabbing to the link and clicking space or enter?